Fix test suite to run locally without errors as well as in the evening (#1238)
cds-snc/notification-api: README.md

`make test`

  * Ensure vars:

>     * SQLALCHEMY_DATABASE_READER_URI
>     * SQLALCHEMY_DATABASE_URI

Reviewer: ```suggestion
    * `SQLALCHEMY_DATABASE_READER_URI`
    * `SQLALCHEMY_DATABASE_URI`
```

>   * are both set either in .env or pytest.ini (precedence goes to .env) to your local db with user access

Reviewer: ```suggestion
  * are both set either in `.env` or `pytest.ini` (precedence goes to `.env`) to your local database with user access
```

>   for CREATE DATABASE access. Additionally the db in the URI should not be set to 'notification_api'.

Reviewer: ```suggestion
  for CREATE DATABASE access. Additionally the database in the URI should not be set to `notification_api`.
```
